massa-execution-worker/src/active_history.rs

```diff
@@ -83,7 +83,9 @@ impl ActiveHistory {
                     return HistorySearchResult::Present(SetUpdateOrDelete::Set(msg));
                 }
                 Some(SetUpdateOrDelete::Update(msg_update)) => {
-                    current_updates.apply(msg_update.clone());
+                    let mut combined_message_update = msg_update.clone();
```

Comment:
I think current_updates does not need to be mut anymore

Reply:
I think it still does because of line 88 :)